plugins/inputs/interrupts/interrupts.go

```diff
@@ -29,7 +35,10 @@ func NewIRQ(id string) *IRQ {
 const sampleConfig = `
   ## To filter which IRQs to collect, make use of tagpass / tagdrop, i.e.
   # [inputs.interrupts.tagdrop]
-    # irq = [ "NET_RX", "TASKLET" ]
+	# irq = [ "NET_RX", "TASKLET" ]
+  #
+  # To report cpus as tags instead of fields use cpus_as_tags
+  # cpus_as_tags = false
```

Comment:
Move this option above the tagdrop usage example, if it is below it will be part of the tagdrop table.  Also in the README.